docs/cudf/source/developer_guide/testing.md

```diff
@@ -0,0 +1,143 @@
+# Testing cuDF
+
+## Tooling
+Tests in cuDF are written using [`pytest`](https://docs.pytest.org/en/latest/).
+Test coverage is measured using [`coverage.py`](https://coverage.readthedocs.io/en/6.4.1/),
+specifically the [`pytest-cov`](https://github.com/pytest-dev/pytest-cov) plugin.
+Code coverage reports are uploaded to [Codecov](https://app.codecov.io/gh/rapidsai/cudf).
+Each PR also indicates whether it increases or decreases in test coverage.
+
+## Test organization
+
+How tests are organized depends on which of the following two groups they fall into:
+
+1. Free functions operating on the above classes like `cudf.merge`.
+2. Methods of classes like `DataFrame` or `Series`.
+
+Tests of free functions should be grouped into files based on the API sections in our documentation.
```

Comment:
(FYI: I’m on a small crusade to reduce the usage of “we/our/you/your” across RAPIDS code and content.)
```suggestion
Tests of free functions should be grouped into files based on the API sections in the documentation.
```